azuredevops/internal/service/serviceendpoint/data_serviceendpoint_azurerm.go

```diff
@@ -0,0 +1,168 @@
+package serviceendpoint
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v6/serviceendpoint"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/client"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/converter"
+)
+
+func DataServiceEndpointAzureRM() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceServiceEndpointAzureRMRead,
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"service_endpoint_name": {
```

Comment:
suggest rename `service_endpoint_name` to `name`

Reply:
renamed the attribute, also added `service_endpoint_name` as a computed value, failing which the `doBaseFlattening` func panics during Acceptance testing
```bash
=== RUN   TestAccServiceEndpointAzureRM_with_serviceEndpointID_DataSource
=== PAUSE TestAccServiceEndpointAzureRM_with_serviceEndpointID_DataSource
=== CONT  TestAccServiceEndpointAzureRM_with_serviceEndpointID_DataSource
panic: Invalid address to set: []string{"service_endpoint_name"}

goroutine 254 [running]:
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*ResourceData).Set(0xc0001b5700, {0xf4aa3b, 0x15}, {0xd864a0, 0xc0006fe540})
        /root/terraform-provider-azuredevops/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource_data.go:230 +0x291
github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/service/serviceendpoint.doBaseFlattening(0xda8080?, 0xc000786bd0, 0xc00082cb60)
```